multicluster/charts/linkerd-multicluster/Chart.yaml

```
@@ -11,7 +11,7 @@ kubeVersion: ">=1.21.0-0"
 name: "linkerd-multicluster"
 sources:
 - https://github.com/linkerd/linkerd2/
-version: 30.11.2-edge
+version: 30.11.3-edge
```

See the other comment I made for viz
```suggestion
version: 30.12.0-edge
```